tla/Traceccfraft.tla

```diff
@@ -108,38 +108,38 @@ TraceInit ==
     \* Constraint the set of initial states to the ones that match the nodes
      \* that are members of the initial configuration
      \* (see  \E c \in SUBSET Servers: ...  in ccraft!InitReconfigurationVars).
-    /\ TraceLog[1].msg.event = [ component |-> "raft", function |-> "add_configuration" ]
-    /\ ToConfigurations(TraceLog[1].msg.configurations) = configurations[TraceLog[1].msg.node]
+    /\ TraceLog[1].msg.function = "add_configuration"
+    /\ ToConfigurations(<<TraceLog[1].msg.new_configuration>>) = configurations[TraceLog[1].msg.state.node_id]
```

Comment:
This does not appear to work, I think my understand of configurations as a list of function here is incorrect. I will have another look tomorrow, but any tips appreciated.

Reply:
Can this be reproduced with the most recent traces2/*.ndjson files?

Reply:
@lemmy no, there was a problem between the keyboard and the chair.